VideoLocker/res/values/dimens.xml

```diff
@@ -59,7 +59,13 @@
     <dimen name="signup_social_title_margin_large">8dp</dimen>
     <dimen name="signup_social_title_margin_bottom">1dp</dimen>
 
+    <dimen name="edx_hairline">1px</dimen>
```

Comment:
I'm pretty sure we want 1px. I'm going to merge for now since this is a small issue but I'll mention it to the designer for confirmation.